cmd/sync/config_test.go

```diff
@@ -49,14 +47,6 @@ func getInvalidConfigInput() []*testInput {
 	invalidRegionCfg.Region = ""
 	input = append(input, &testInput{invalidRegionCfg, "invalid region"})
 
-	invalidUpstreamConfEndponCfg := getValidConfig()
```

Comment:
Add a test for invalid APIEndpoint, similar to the `invalidUpstreamConfEndponCfg`